estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models,fields
+
+class EstateProperty(models.Model):
+    _name= 'estate.property'
+    _description= 'This is a estate property app'
+
+    def set_selection(self):
+        return 'north'
```

Comment:
```suggestion
    def set_selection(self):
        return 'north'
```
This is correct way of defining default function but for now you can directly assign value to default, we will use function when you need to perform some operation to find and than assign the value